### DISCUSSION TITLE
ESLint: More rules for the new Script component

### DISCUSSION THREAD

**Discussion by awareness481** [vercel/next.js#26365] [Ideas]
Basically, disallow `next/script` imports in `_document.js` and disallow `<Script />` being a child of `<Head>`. Based on this: https://github.com/vercel/next.js/pull/26253/commits/f4aab83d37428f15994080ca8b00c46d77d78231

I've already got an initial implementation working locally, let me know if this is something you want to see added.

**Comment by awareness481** [comment]
@housseindjirdeh Have a bit of free time until the end of week. Would love to implement this if this is something you folks want.

**Comment by housseindjirdeh** [comment]
Both rules sound great, please feel free to submit a PR @awareness481 :)

CC @janicklas-ralph for a sanity check
